A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU.css

```diff
@@ -102,22 +105,22 @@ a320-neo-cdu-main-display::after {
 }
 */
 .white {
-  color: white; }
+  color: var(--displayWhite); }
 
 .blue {
-  color: #00ffff; }
+  color: var(--displayCyan); }
 
 .yellow {
-  color: #f9dd02; }
+  color: var(--displayYellow); }
 
 .green {
-  color: #00ff00; }
+  color: var(--displayGreen); }
 
 .red {
-  color: #ff9a00; }
+  color: var(--displayRed); }
 
 .magenta {
-  color: #f800cc; }
+  color: var(--displayMagenta); }
```

Comment:
I don't think these should have been changed, we already fixed these colors in #888.

Reply:
Reverted them to the original colors, put the color definitions in the new common CSS file.